assets/css/post-entry.css

```diff
@@ -104,3 +104,7 @@
     color: var(--secondary);
     box-shadow: 0 1px 0 var(--primary)
 }
+
+.entry-cover p {
+	margin: 8px 0 16px;
+}
```

Comment:
Cover seems fine to me. Can you please remove this

Reply:
I had added this to make the cover image captions have the same padding as  the other image captions. I'm fine with removing it, but are you sure we don't want this?

**With padding:**
![caption with padding](https://user-images.githubusercontent.com/63574107/108542072-1ce07680-7298-11eb-961c-e3b38a528f2f.png)

**Without padding:**
![caption without padding](https://user-images.githubusercontent.com/63574107/108542105-29fd6580-7298-11eb-8650-6b6d5f2a2e30.png)

Reply:
yes

Reply:
I removed the second commit. Should be good to go now.